symengine/printer.cpp

```diff
@@ -414,30 +416,34 @@ void uintpoly_print(const T &x, std::ostringstream &s)
     }
     if (x.size() == 0)
         s << "0";
+    return s.str();
 }
 
 void StrPrinter::bvisit(const UIntPoly &x)
 {
-    std::ostringstream s;
-    uintpoly_print(x, s);
-    str_ = s.str();
+    str_ = upoly_print<UIntPoly>(x);
```

Comment:
I will actually have to shift the `upoly_print` code to the `.h` file? Should I go ahead?